build.gradle

```diff
@@ -17,7 +17,7 @@ ext {
 		mockito                     : "4.3.1",
 		glassfish_version           : "3.0.3",
 		powertools_version          : "1.12.3",
-		cri_common_lib              : "1.1.1",
+		cri_common_lib              : "1.1.2",
```

Comment:
we may as well refer to version 1.1.3 here or potentially 1.1.4 if it is published prior to this PR